### ISSUE TITLE
Require images on submission

### ISSUE THREAD

**Issue by cheerfulstoic** [neo4j-examples/graphgist_portal#46]

**Comment by jexp** [comment]
Also description

**Comment by whatSocks** [comment]
It would be a time saver to pull the first (?) image from the graph gist as the default image

**Comment by cheerfulstoic** [comment]
True, that would be a nice option for the submission form.  I've done that in the past to populate graphgists that don't have images.  Not all graphgists will have images in them, though

**Comment by cheerfulstoic** [comment]
Just thinking about it now, it actually might be tricky because when we preview we load the preview in an iframe which wouldn't allow us to show the images in the submission form...
